tiles/test/index.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { createExecutionContext, env, SELF, waitOnExecutionContext } from 'cloudflare:test';
+import { inject } from 'vitest';
+import worker, { parseUrl } from '../src';
+
+// For now, you'll need to do something like this to get a correctly-typed
+// `Request` to pass to `worker.fetch()`.
+const IncomingRequest = Request<unknown, IncomingRequestCfProperties>;
+
+describe('integration tests', () => {
+  describe('success - zoom1', () => {
+    beforeAll(async () => {
+      const file = await fetch(`http://localhost:${inject('port')}/zoom1.pmtiles`);
+      if (!file.body) throw new Error('File body is undefined');
+      const body = file.body as ReadableStream;
+      await env.BUCKET.put(env.PMTILES_FILE_NAME, body);
+      console.log('File uploaded');
+    }, 30000);
+    it('responds with correct json file', async () => {
+      const response = await SELF.fetch('https://example.com/v1');
+      expect(JSON.stringify(await response.json(), null, 2)).toMatchFileSnapshot('./__snapshots__/v1.json');
+    });
+    it('responds with correct tile', async () => {
+      const response = await SELF.fetch('https://example.com/v1/0/0/0.mvt');
+      expect(response.status).toBe(200);
+      expect(Buffer.from(await response.arrayBuffer()).toString()).toMatchFileSnapshot('./__snapshots__/v1-0-0-0.mvt');
+    });
+    it('responds with correct tile 2', async () => {
+      const response = await SELF.fetch('https://example.com/v1/1/0/1.mvt');
+      expect(response.status).toBe(200);
+      expect(Buffer.from(await response.arrayBuffer()).toString()).toMatchFileSnapshot('./__snapshots__/v1-1-0-1.mvt');
+    });
+    it('responds with error when tile out of bounds', async () => {
+      const response = await SELF.fetch('https://example.com/v1/0/0/1.mvt');
+      expect(response.status).toBe(500);
+    });
+  });
+});
+
+describe.skip('Hello World worker', () => {
```

Comment:
do these still not work/need to be skipped?

Reply:
Not added those unit tests yet